From 7b0e08e0da9f3fac3d65fffae25f3617b40531c0 Mon Sep 17 00:00:00 2001
From: joooooon9 <wjdwns9988@naver.com>
Date: Wed, 3 Nov 2021 22:32:17 +0900
Subject: [PATCH] =?UTF-8?q?=EC=B2=B4=EB=A6=AC=ED=94=BD=EC=BB=A4=EB=B0=8B?=
 =?UTF-8?q?=EC=B6=94=EA=B0=80?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
 cherrtpick.md | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cherrtpick.md b/cherrtpick.md
index e69de29..5222f23 100644
--- a/cherrtpick.md
+++ b/cherrtpick.md
@@ -0,0 +1 @@
+체리픽커밋추가
-- 
2.33.0.windows.2

